configuration/postgres/postgres.go

```diff
@@ -152,55 +179,51 @@ func (p *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequ
 	if err != nil {
 		return nil, fmt.Errorf("unable to parse response from configuration store - %w", err)
 	}
-	result := make(map[string]*configuration.Item)
-	for _, v := range items {
-		result[v.key] = v.item
-	}
+	result := getUniqueItemPerKey(items)
```

Comment:
This is to return the `latest` version of the key if there are multiple versions present in the config store as per discussed in https://github.com/dapr/components-contrib/issues/2187
cc @ItalyPaleAle